src/components/ProductCard/ProductCard.js

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components/macro'
+
+function ProductCard({ name, description, image }) {
+  return (
+    <ProductCardWrapper>
+      <ProductImg src={image} alt={`${name} image`} />
```

Comment:
The `alt` text is not really perfect. Would be better to just leave it empty as the `name` is already describing the image? So, right now if the name is "Product One" a screenreader would read your card as: Image Product One image Product One ...
We can get rid of the `alt`-text if the information is conveyed through the rest of the card. 
```suggestion
      <ProductImg src={image} alt="" />
```